tests/framework/utils/helm_helper.go

```diff
@@ -55,12 +56,23 @@ func (h *HelmHelper) InstallLocalRookHelmChart(chartName string, deployName stri
 	if namespace != "" {
 		cmdArgs = append(cmdArgs, "--namespace", namespace)
 	}
-	_, err := h.Execute(cmdArgs...)
+	result, err := h.Execute(cmdArgs...)
 	if err != nil {
-		logger.Errorf("cannot install helm chart with name : %v, version: %v, namespace: %v , err: %v", chartName, chartVersion, namespace, err)
-		return fmt.Errorf("cannot install helm chart with name : %v, version: %v, namespace: %v , err: %v", chartName, chartVersion, namespace, err)
-	}
+		logger.Infof("Initial helm install failed, will retry in 10 sec")
+		time.Sleep(time.Second * 10)
+		ls, _ := h.Execute([]string{"ls"}...)
```

Comment:
why not always execute these commands such as `helm ls` and `helm search` and print at info level? It could be useful in the log.

Reply:
that's a good idea, I will print them at Info.